package.json

```diff
@@ -8,7 +8,10 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./lib/index.js"
+      "import": "./lib/herumi/index.js",
+      "node": "./lib/blst-native/index.js",
+      "browser": "./lib/herumi/web.js",
+      "bun": "./lib/herumi/index.js"
```

Comment:
isn't one of the promises of bun that you can run any nodejs app?  If we having a different import for it kinda defeats that

Reply:
It is, now their napi support is still a bit fragile. They are making fast progress though, so I see this more as a short term stop gap measure.

Reply:
> It is, now their napi support is still a bit fragile

why do we want to support bun so badly, are there any known consumers that want this?

For browser I can see that the conditional export makes sense as you avoid to load the node code that way and can reduce the bundle size.

For bun, it might be better to check if native api is available and load it based on that. Then you would at least be able to run the same code (without modifying imports) in both node and bun.

Reply:
so as I understand, bun would work now as well since it always falls back to herumi

Reply:
Unfortunately when trying to load native bun crashes. No exception are thrown.
I didn't try with latest bun, maybe it's better now.